packages/reactotron-core-client/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "noImplicitAny": false, // Might be worth setting to true after we get good typing in
```

Comment:
I am open to that. I was just trying to go the path of least resistance here so I could test if the output was actually able to be dropped in and used.